recipe/meta.yaml

```diff
@@ -56,10 +65,18 @@ outputs:
 about:
   home: https://github.com/slackapi/python-slack-sdk
```

Comment:
The slack.dev URL appears to be the "official" home page.

Reply:
Thanks. I've updated it